dashboard/lib/pd/payment/payment_factory.rb

```diff
@@ -27,7 +27,7 @@ def self.get_calculator_class(workshop)
         ].include?(workshop.course)
       end
 
-      nil
+      PaymentCalculatorUnpaid
```

Comment:
Is there a test for this that verifies that paymentcalculatorunpaid is what it falls back to?